app/models/work.rb

```diff
@@ -0,0 +1,123 @@
+class Work < ApplicationRecord
+  validates :category, presence: true
+  validates :title, presence: true, uniqueness: {scope: :category}
+  validates_length_of :title, minimum: 1, maximum: 150
+  validates :creator, presence: true
+  validates_length_of :creator, minimum: 1, maximum: 150
+  validates :description, presence: true
+  validates_length_of :description, minimum: 1, maximum: 350
+  validates :publication_year, presence: true
+  validates :vote_count, presence: true
```

Comment:
I realize why you have `vote_count`, as it makes an easy way to sort works.  However you would need some way to ensure that it's synced to the length of the `.votes` collection.  

Better to use the relationship to get the count.  There are ways to get the works in order using that relationship.